public/swagger.v1.json

```diff
@@ -232,6 +232,14 @@
             "description": "Limit of result\n\nmaximum: setting.ExplorePagingNum",
             "name": "limit",
             "in": "query"
+          },
```

Comment:
Do not add swagger stuff manually because it will be overwritten if you run `make generate-swagger`.